doc/user/content/releases/previews.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Previews"
+description: ""
+disable_list: true
+menu:
+  main:
+    parent: releases-previews
+    weight: 50
+    identifier: previews
+---
+
+The features listed below are available as [private/public
+preview](https://materialize.com/preview-terms/). Public preview features are
+enabled by default; private previews can be enabled for your region(s). If you
+would like to enable a private preview feature for your region(s), contact your
+Materialize support.
+
+{{< note >}}
+
+- The features are under **active development** and may have stability or
+  performance issues.
+
+- The features are **not** subject to our backwards compatibility guarantees.
+
+{{</ note >}}
+
+## Private Preview
+
+### Health Dashboard
```

Comment:
Notion mentions some other previews in the Console.  But, at least, as a starting point, we'll go with the health dashboard.